test/examples/positionGlobe.js

```diff
@@ -37,7 +38,7 @@ describe('positionGlobe', () => {
                 globeView.notifyChange(true);
             }));
 
-        await page.screenshot({ path: '/tmp/b.png' });
+        await page.screenshot({ path: `${os.tmpdir()}/b.png` });
```

Comment:
By the way, I've just realized that this screenshot was a debug code that I forgot to remove before merging the PR :)

Could you remove it completely instead of fixing it?

Reply:
removed